omod/src/main/webapp/resources/htmlforms/hiv/moh257VisitSummary.html

```diff
@@ -760,10 +760,10 @@
         });
     </script>
 
-    <!-- For now HFE requires encounter provider, so we set this to the "Unknown Provider" and hide it (see HTML-471) -->
-    <div style="display:none">
-        <encounterProvider default="ae01b8ff-a4cc-4012-bcf7-72359e852e14" />
-    </div>
+    <!--&lt;!&ndash; For now HFE requires encounter provider, so we set this to the "Unknown Provider" and hide it (see HTML-471) &ndash;&gt;-->
+    <!--<div style="display:none">-->
+        <!--<encounterProvider default="ae01b8ff-a4cc-4012-bcf7-72359e852e14" />-->
+    <!--</div>-->
```

Comment:
Can you add some documentation for commenting out. It will act as a guide to those who will be wondering why it was commented out